sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallMedia.java

```diff
@@ -87,23 +87,23 @@ public Response<Void> playToAllWithResponse(PlaySource playSource, PlayOptions o
 
     /**
      * Recognize tones.
-     * @param recognizeOptions Optional elements for recognize.
+     * @param callMediaRecognizeOptions Optional elements for recognize.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public void recognize(RecognizeOptions recognizeOptions) {
-        callMediaAsync.recognize(recognizeOptions).block();
+    public void recognize(CallMediaRecognizeOptions callMediaRecognizeOptions) {
```

Comment:
Do we need the changes in `callingserver` anymore?

Reply:
Please remove any changes in callingserver. It is deprecated and I will request to delete the code and pipeline soon